### PR TITLE
Update to .NET Core RTM

### PR DESCRIPTION
**Pull request by clarkis117**
#91 Updated the library, tests, and website to .NET Core RTM. Everything should be working as designed. In addition, I incremented the version numbers for GenFu in the project.json files to 1.1.2 to reflect the changes made.

### CODE REVIEW

File: src/GenFu.Web/Models/SourceCode.cs [hunk 1]
```diff
@@ -9,52 +9,45 @@
...
 
-    public class SourceCode
-    {
 		// todo: make not hacky
 		public AssemblyLoadContext Accessor { get; set; }
 
-		public ILibraryExporter LibraryExporter { get; set; }
+		//cache meta data, these bits are costly to load and can cause frequent GC
+		//also since we have a hard coded list there's no need to recompute this per request
+		private static readonly List<MetadataReference> references = new List<MetadataReference>();
```
Comment (clarkis117) [inline src/GenFu.Web/Models/SourceCode.cs]: One particular item to note in the reworking of the code compilation, I choose to cache MetadataReferences as static variable (list) as they’re expensive to load per request and don’t change between requests. Reloading the references per each request causes significant heap growth to the point where the GC is running per each request.

**Comment by FabienDehopre** [comment]
Nice job!
Now, it'd be really nice if this pull request could be merged and then published on nuget ASAP.

**Comment by dpaquette** [comment]
Reviewing this now

File: src/GenFu/project.json [hunk 2]
```diff
@@ -1,33 +1,30 @@
...
+	"buildOptions": {
+		"embed": "Resources/*.*"
+	},
+
+	"dependencies": {
+			"NETStandard.Library": "1.6.0"
+	},
+
+	"frameworks": {
+		"netstandard1.6": {},
```
Comment (dpaquette) [inline src/GenFu/project.json]: I wonder if we can support a lower version of netstandard to target more platforms. We'll do as a separate issue

**Comment by dpaquette** [comment]
Great work @clarkis117 ! Thanks for doing this

File: src/GenFu/project.json (hunk 2 above)
Comment (clarkis117) [inline src/GenFu/project.json]: I got it down to netstandard1.3  lower than that and the System.Reflection.TypeExtensions doesn't support it.

**Pull request merged**